doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -336,24 +362,38 @@ You can see the `DCGAN script <https://github.com/ray-project/ray/blob/master/py
 Custom Training and Validation Functions
 ----------------------------------------
 
+.. warning:: This is still an experimental API and is subject to change in the near future.
+
 ``PyTorchTrainer`` allows you to run a custom training and validation step in parallel on each worker, providing a flexibility similar to using PyTorch natively. This is done via the ``train_function`` and ``validation_function`` parameters.
 
-Note that this is needed if the model creator returns multiple models.
+Note that this is needed if the model creator returns multiple models, optimizers, or schedulers.
 
 .. code-block:: python
 
-    def train(models, dataloader, criterion, optimizers, config):
-        """A custom training function.
+    # An example custom training function.
+
```

Comment:
remove newline?

Reply:
also consider removing the comment, doesn't add much on top of the name of the section

Reply:
done